manifests/d/DoltHub/Dolt/1.5.0/DoltHub.Dolt.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: DoltHub.Dolt
+PackageVersion: 1.5.0
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+Scope: machine
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: Dolt 1.2.4
+  UpgradeCode: '{6BC40754-759D-4899-9FD3-E6BE1823CACD}'

```

Comment:
```suggestion
```
We can get rid of those as these since Publisher and PackageName are already a good match. These require manual editing for each update